From ec54b1efe74ca4b6cc1e432046de6269029c02ed Mon Sep 17 00:00:00 2001
From: Heiko Becker <mail@heiko-becker.de>
Date: Thu, 27 Feb 2025 23:33:07 +0100
Subject: [PATCH] Fix build with Qt 6.9

That version added some overloads to setBrush [1], making the {}
ambiguous:
"hotspot/src/models/timelinedelegate.cpp:197:26: error: call of overloaded
'setBrush(<brace-enclosed initializer list>)' is ambiguous"

[1] qtbase commit: 2ad79c626d7a94e545886152ce5ac3feabf619a2
---
 src/models/timelinedelegate.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/models/timelinedelegate.cpp b/src/models/timelinedelegate.cpp
index b567f280..e666d122 100644
--- a/src/models/timelinedelegate.cpp
+++ b/src/models/timelinedelegate.cpp
@@ -194,7 +194,7 @@ void TimeLineDelegate::paint(QPainter* painter, const QStyleOptionViewItem& opti
         painter->drawRect(threadTimeRect.adjusted(-1, -1, 0, 0));
 
         // visualize all events
-        painter->setBrush({});
+        painter->setBrush(QBrush());
 
         if (offCpuCostId != -1) {
             const auto offCpuColor = scheme.background(KColorScheme::NegativeBackground).color();
